datatypes.go

```diff
@@ -6,6 +6,7 @@ import (
 	"math"
 	"strconv"
 	"strings"
+	"text/template"
```

Comment:
I think that we need to use `html/template` instead of `text/template`

Reply:
Why?

Reply:
Official documentation for `html/template` states that:

> Package template (html/template) implements data-driven templates for generating HTML output safe against code injection. It provides the same interface as package text/template and should be used instead of text/template whenever the output is HTML.

So we need to choose `html/template` if there is a risk of HTML injection in description field using our templating mechanism.